tools/licenses/lib/filesystem.dart

```diff
@@ -257,6 +257,7 @@ FileType identifyFile(String name, Reader reader) {
     case '.diff':
       // Don't try to read the copyright out of patch files, since there'll be fragments.
       return FileType.binary;
+    case '.3':
```

Comment:
i think we do .1 in the `paths.dart` file, fwiw. might be worth making the two consistent with each other.

Reply:
skipping entirely (i.e. doing it in `paths.dart`) may be more correct if we don't ship the man file (which i assume we don't).

Reply:
Ah, interesting. Thanks, done.